contracts/src/interfaces/IIPTokenStaking.sol

```diff
@@ -23,7 +23,7 @@ interface IIPTokenStaking {
     /// @param shortStakingPeriod The staking duration for short staking, in seconds
     /// @param mediumStakingPeriod The staking duration for medium staking, in seconds
     /// @param longStakingPeriod The staking duration for long staking, in seconds
-    /// @param unjailFee The fee required to unjail a validator
+    /// @param validatorUpdateFee The fee required to unjail a validator
```

Comment:
Is this the fee that required to update validator info? including unjail. If so, let's update comment as well.

Reply:
Refactoring to "fee" so it applies to operator methods as well